pkg/image/filefmt.go

```diff
@@ -67,10 +66,12 @@ func CopyKnownHdrs() Headers {
 	return m
 }
 
+// Match performs a check to see if the provided byte array matches the bytes in our header data
 func (h Header) Match(b []byte) bool {
 	return bytes.Equal(b[h.mgOffset:h.mgOffset+len(h.magicNumber)], h.magicNumber)
 }
 
+// Size calculates the size of the provided byte array as an int64 and returns it or an error
```

Comment:
Not really accurate. `Size` extracts from the receiver the byte size of the original file represented by this header's metadata.

Reply:
Thanks, like I said; there wasn't a road map or any comments so I pieced these together best I could; I'll see if I can fix that up.